app/logic/fileHandler.py

```diff
@@ -49,11 +58,27 @@ def saveFiles(self, saveOriginalFile=None):
                     if not isFileInCourse:
                         AttachmentUpload.create(course=self.courseId, fileName=file.filename)
                         saveFileToFilesystem = file.filename
+                elif self.programId:
+
+                    # remove the existing file
+                    deleteFileObject = AttachmentUpload.get_or_none(program=self.programId)
+                    if deleteFileObject:
+                        self.deleteFile(deleteFileObject.id)
+
+                    # add the new file
+                    fileType = file.filename.split('.')[-1]
+                    fileName = f"{self.programId}.{fileType}"
+                    AttachmentUpload.create(program=self.programId, fileName=fileName)
+                    current_programID = fileName
+                    saveFileToFilesystem = current_programID
+
```

Comment:
I think using snake_case for variable naming is better. 
`deleteFileObject `to `delete_file_object.`
`fileType` to `file_type.`
`fileName` to `file_name.`

Reply:
I remember we discussed this in-person last week. The code base often uses camelCase instead of snake case.